gcloud/bigtable/table.py

```diff
@@ -95,20 +97,38 @@ def column_family(self, column_family_id, gc_rule=None):
         """
         return ColumnFamily(column_family_id, self, gc_rule=gc_rule)
 
-    def row(self, row_key, filter_=None):
+    def row(self, row_key, filter_=None, append=False):
         """Factory to create a row associated with this table.
 
+        .. warning::
+
+           At most one of ``filter_`` and ``append`` can be used in a
+           :class:`Row`.
+
         :type row_key: bytes
         :param row_key: The key for the row being created.
 
         :type filter_: :class:`.RowFilter`
         :param filter_: (Optional) Filter to be used for conditional mutations.
-                        See :class:`.Row` for more details.
+                        See :class:`.DirectRow` for more details.
+
+        :type append: bool
+        :param append: (Optional) Flag to determine if the row should be used
+                       for append mutations.
 
-        :rtype: :class:`.Row`
+        :rtype: :class:`.DirectRow`
```

Comment:
`AppendRow` doesn't derive from `DirectRow`, which makes `Row` the only signular choice.  You could signal "one of `DirectRow`, `ConditionalRow`, or `AppendRow`, maybe,